src/Cli/dotnet/commands/dotnet-workload/update/WorkloadUpdateCommand.cs

```diff
@@ -124,7 +124,7 @@ public void UpdateWorkloads(bool includePreviews = false, DirectoryPath? offline
             _reporter.WriteLine();
             var featureBand = new SdkFeatureBand(string.Join('.', _sdkVersion.Major, _sdkVersion.Minor, _sdkVersion.SdkFeatureBand));
 
-            var workloadIds = _workloadInstaller.GetWorkloadInstallationRecordRepository().GetInstalledWorkloads(featureBand);
+            var workloadIds = GetUpdatableWorkloads();
             _workloadManifestUpdater.UpdateAdvertisingManifestsAsync(includePreviews, offlineCache).Wait();
             var manifestsToUpdate = _workloadManifestUpdater.CalculateManifestUpdates();
```

Comment:
Should advertising manifest use the high feature band too? We should use advertising manifest at all?

Reply:
Just double checked, the WorkloadManifestUpdater uses whatever feature band is used by the manifest provider, which is always the current version/ version provided the SDK version option, which should be correct here.